solution/ui/regulations/eregs-vite/src/components/SearchEmptyState.stories.js

```diff
@@ -0,0 +1,21 @@
+import SearchEmptyState from "./SearchEmptyState.vue";
+
+export default {
+    title: "Regulations/Search/Empty State",
+    component: SearchEmptyState,
+};
+
+const Template = (args, { argTypes }) => ({
+    props: Object.keys(argTypes),
+    components: { SearchEmptyState },
+    template: '<SearchEmptyState v-bind="$props" />',
+});
+
+export const Basic = Template.bind({});
+Basic.args = {
+    eregs_url: "https://www.google.com",
```

Comment:
Should this be google?  Do we not have the proper url?

Reply:
This URL can either be for the resources page or the regulations search.  What I should probably do is use a custom control to let storybook users simply pick between those two options.